hooks/hookstage/bidderrequest_mutations.go

```diff
@@ -31,12 +31,12 @@ func (c ChangeSetBidderRequest[T]) BApp() ChangeSetBApp[T] {
 	return ChangeSetBApp[T]{changeSetBidderRequest: c}
 }
 
-func (c ChangeSetBidderRequest[T]) castPayload(p T) (*openrtb2.BidRequest, error) {
+func (c ChangeSetBidderRequest[T]) castPayload(p T) (*openrtb_ext.RequestWrapper, error) {
 	if payload, ok := any(p).(BidderRequestPayload); ok {
-		if payload.BidRequest == nil {
+		if payload.RequestWrapper == nil {
 			return nil, errors.New("empty BidRequest provided")
```

Comment:
Nitpick: We should update the error messages in this method to not refer to the specific argument. Perhaps messages like "payload contains a nil bid request"  and "failed to cast payload".